projects/packages/search/.gitignore

```diff
@@ -1,4 +1,4 @@
 .cache/
-build/
+dist/
```

Comment:
Can I persuade you to still use `tools`and `build`? - they are definitely not the best names but I guess it's better to be consistent with other parts of the repo.

Reply:
You make an excellent point here. I'll go ahead and start reverting some of these changes and instead use `build` and `tools`.